bundler/spec/install/gems/native_extensions_spec.rb

```diff
@@ -87,6 +87,58 @@
     expect(out).to eq("true")
   end
 
+  it "installs from git when multiple gems share one repository" do
```

Comment:
To make the wording of the spec more exact, I'd say:

```suggestion
  it "installs correctly from git when multiple gems with extensions share one repository" do
```

Just pointing out that the test is about extensions, and adding "correctly" to try to emphasize that the install won't actually fail, but will install "incorrectly" (extensions artifacts overwriting each other).